Jenkinsfile

```diff
@@ -5,19 +5,37 @@ pipeline {
             steps {
                 sh 'hatch run docs:build'
             }
+
+	    post {
+      		 success {
+        	  publishHTML([allowMissing: false,
+                     alwaysLinkToLastBuild: false,
+                     keepAll: true,
+                     reportDir: 'docs/site',
+                     reportFiles: 'index.html',
+                     includes: '**/*',
+                     reportName: 'Cycax Docs',
+                     reportTitles: 'Cycax Docs report'
+                  ])
+                }
+            }	
+        }
+	stage('Mypy') {
+	    steps {
+		sh (label: 'Run MyPy',
```

Comment:
You are mixing styles here, stick with sh, consistency will help to debug these tasks.
```
sh 'mkdir -p reports'
sh 'hatch run lint:typing |& reports/mypy.txt'
```

I suggest you do not call it logs. Small difference but these are record that we process not just logs.
We should be consistent in all the repos. Whatever name we pick we should use it in all repos.